resources/assets/js/markdown-editor/css/style.css

```diff
@@ -45,10 +45,22 @@
     @apply h-auto px-4 py-2 text-base font-bold text-center transition-all rounded;
 }
 .ark-markdown-editor .tui-popup-body .te-button-section button.te-ok-button {
-    @apply ml-3 text-white bg-theme-primary-600;
+    @apply ml-3 text-white bg-theme-primary-600 px-4 py-2.5 font-semibold text-center rounded;
+    transition: all 0.1s ease-in;
```

Comment:
can this be the `transition-default` class instead? 🤔

Reply:
I noticed it doesn't find the `transition-default` class. I suppose it's due to being included before we define that custom class. Do you think we should make the editor styling an explicit import so we can define it in the project's general css file? Would still have to test if that fixes the issue though

Reply:
@ItsANameToo the markdown styles are just 10kb (uncompressed) think is not that bad and another advantage would be that we can rollback [these changes](https://github.com/ArkEcosystem/marketsquare.io/pull/1518) also I would be able to use `button-primary` instead of all those classes

The only downside is that will be added to all projects but think it has more advantages than disadvantages.

Reply:
> The only downside is that will be added to all projects 

why? can't we make it something a project imports if they need it? plus if you remove the `/* purgecss start ignore */` line and make it something that the project should define in their tailwind config it will just be purged if unused

Reply:
@ItsANameToo I suppose I can add the plugin JS files related to our custom component to the purgecss settings, and __in theory__ it should work, need to make some tests. Still, we will need to modify the purgecss settings for the projects that use this library to ignore/add the markdown editor classes.

Reply:
yeah it will be needed to adjust the purge settings in a project, but that shouldn't be an issue. We already do that for some styles like the blog styling as that's not relevant to all projects